guides/source/5_0_relase_notes.md

```diff
@@ -0,0 +1,198 @@
+**DO NOT READ THIS FILE ON GITHUB, GUIDES ARE PUBLISHED ON http://guides.rubyonrails.org.**
+
+Ruby on Rails 5.0 Release Notes
+===============================
+
+Highlights in Rails 5.0:
+
+* Ruby 2.2.2+ required
+* ActionCable
+* API Mode
+* Exclusive use of `rails` CLI over Rake
+* Sprockets 3
+* Turbolinks 5
+
+These release notes cover only the major changes. To learn about various bug
+fixes and changes, please refer to the change logs or check out the [list of
+commits](https://github.com/rails/rails/commits/5-0-stable) in the main Rails
+repository on GitHub.
+
+--------------------------------------------------------------------------------
+
+Upgrading to Rails 5.0
+----------------------
+
+If you're upgrading an existing application, it's a great idea to have good test coverage before going in. You should also first upgrade to Rails 4.2 in case you haven't and make sure your application still runs as expected before attempting an update to Rails 5.0. A list of things to watch out for when upgrading is available in the [Upgrading Ruby on Rails](upgrading_ruby_on_rails.html#upgrading-from-rails-4-2-to-rails-5-0) guide.
+
+
+Creating a Rails 5.0 application
```

Comment:
Can we remove this section? I think this was added to the release notes when Rails fully adopted bundler (and maybe in a few releases after that). Bundler has been around for a long enough time.